api/make_listener_posix.go

```diff
@@ -48,7 +49,8 @@ func makeListener(cfg Config) (net.Listener, error) {
 
 	if network == unixNetwork {
 		if _, err := os.Stat(path); !os.IsNotExist(err) {
```

Comment:
How about removing the `Stat` call as well?  Then we remove the whole pattern that resulted in the bug.

Reply:
since we don't use the stat info then yes that will work, thus I pushed it here 715ce2bb209014fdbf6ee0a31d0b4eddb41bdad6